client/src/modules-data.json

```diff
@@ -0,0 +1,176 @@
+{
+  "modules": [
+    {
+      "id": 1,
+      "module_name": "Html-Css",
+      "topics": [
+        {
+          "topic_id": 1,
+          "topicName": "Fundamentals • Semantics",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 2,
+          "topicName": "Forms • Structuring Data",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 3,
+          "topicName": "Layout • Flex box • Grid",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 3,
+          "topicName": "Layout • Flex box • Grid",
+          "topicRef": "www.HTML-CSS.com"
+        }
+      ]
+    },
+    {
+      "id": 2,
+      "module_name": "JavaScript_1",
+      "topics": [
+        {
+          "topic_id": 1,
+          "topicName": "Hello world (of jS)",
+          "topicRef": "www.JavaScript.com"
+        },
+        {
+          "topic_id": 2,
+          "topicName": "Logic and Loops",
+          "topicRef": "www.JavaScript.com"
+        },
+        {
+          "topic_id": 3,
+          "topicName": "Logic and Arrays",
+          "topicRef": "www.JavaScript.com"
+        },
+        {
+          "topic_id": 4,
+          "topicName": "Array methods and higher order functions",
+          "topicRef": "www.JavaScript.com"
+        }
+      ]
+    },
+    {
```

Comment:
Question: 
How do we plan to model this data in our database?

Reply:
Done